tools/cockpit.spec

```diff
@@ -454,11 +454,16 @@ Requires: glib2 >= 2.50.0
 Conflicts: firewalld < 0.6.0-1
 Recommends: sscg >= 2.3
 Recommends: system-logos
-Requires: systemd >= 235
 Suggests: sssd-dbus
+%if !0%{?suse_version}
+Requires: systemd >= 235
```

Comment:
We need that in Fedora/Debian/etc. to get proper values for 

    systemdunitdir=$($PKG_CONFIG systemd --variable=systemdsystemunitdir)

how does that work in OpenSUSE? is systemd.pc shipped in a different package? I'd also be okay with generally dropping that build dependency and hardcoding `--with-systemdunitdir=/usr/lib/systemd/`, but it feels wrong to do that. What's the problem with this extra build dep?

Reply:
This one is about the run time requirement. The BuildRequires stays of course :)

Reply:
Oh, right. systemd is a hard requirement for cockpit.socket/cockpit.service, but I see the container build use case (we actually do that upstream as well, https://hub.docker.com/r/cockpit/ws/).

The main point of this was to make sure that it's not running against a too old systemd, but all our supported OSes now have a sufficiently new one. So it's fine for me to simply drop this Requires: completely.

Reply:
Ok. In such a case a conflicts with < someversion would also work.